core/templates/pages/exploration-editor-page/settings-tab/settings-tab.component.spec.ts

```diff
@@ -39,6 +39,18 @@ import { WindowRef } from 'services/contextual/window-ref.service';
 import { UserExplorationPermissionsService } from
   'pages/exploration-editor-page/services/user-exploration-permissions.service';
 import { HttpClientTestingModule } from '@angular/common/http/testing';
+import { EventEmitter } from '@angular/core';
```

Comment:
Ditto, keep existing organization and group this with the other `@angular` import.

Reply:
Done